libraries/AP_UAVCAN/AP_UAVCAN.cpp

```diff
@@ -1035,4 +1197,96 @@ void AP_UAVCAN::update_mag_state(uint8_t node)
     }
 }
 
+//EFI
+uint8_t AP_UAVCAN::register_efi_listener(AP_EFI_Backend* new_listener, uint8_t preferred_channel)
+{
+    uint8_t sel_place = 255, ret = 0;
```

Comment:
please use UINT8_MAX instead of 255 to match the rest of the drivers. This may have been a change since you opened your PR